src/public/WRLDMapView.h

```diff
@@ -236,6 +238,24 @@ extern NSString * const WRLDMapViewNotificationCurrentFloorIndex;
 - (void)removePolygons:(NSArray <WRLDPolygon *> *)polygons;
 
 
+#pragma mark - building highlights -
+
+/*! @name Building highlights */
+
+/*!
+ Create a building highlight and add it to the map.
+ @param buildingHighlightOptions Creation parameters for the building highlight.
+
+ @return The WRLDBuildingHighlight that was added.
+ */
+- (WRLDBuildingHighlight*) addBuildingHighlight:(WRLDBuildingHighlightOptions*) buildingHighlightOptions;
```

Comment:
Is signature appropriate? Elsewhere on this interface we have "add" methods with param of the instance of object that we are adding, with void return, eg addPolygon, addPositioner
https://github.com/wrld3d/ios-api/blob/master/src/public/WRLDMapView.h#L215
https://github.com/wrld3d/ios-api/blob/master/src/public/WRLDMapView.h#L215

This methods seems more of a "createAndAdd", both a factory and an api to add to view.